lib/Game_Logic/gamelogic.cpp

```diff
@@ -285,4 +315,9 @@ void gamelogic::updateScore()
 {
     currentScore += currentGrabbedItem->value;
     scoreHasChanged = true;
+}
+
+bool gamelogic::isEndOfRound(int seconds, int startTimeRound){
```

Comment:
is end of round sounds like a boolean the function should be something like check end of round the returned boolean can me is end of round